### PR TITLE
analysis-preservation-bootcamp-post

### PR DESCRIPTION
**Pull request by cranmer**
I developed this locally and it builds fine and layout is fine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,9 @@ GitHub pages for IRIS-HEP website
 Do not make a PR to the output repository directly! Instead, make pull requests to the [iris-hep.github.io-source repository](https://github.com/iris-hep/iris-hep.github.io-source/). The advantage of making a pull request instead of directly editing is that changes made via pull request will be tested to ensure that all is ok with your changes...
 
 
+## testing locally:
+
+```
+gem install bundler jekyll
```
Comment (henryiii) [inline README.md:15]:
```suggestion
gem install bundler
```

You shouldn't install Jekyll here. Just bundler, then bundler will handle the rest.

File: _posts/2020-02-17-analysis-preservation.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ * Rokas Maciulaitis
+ * Diego Rodriguez Rodriguez
+ * Marco Vidal
+
+
+
+<!--<blockquote class="twitter-tweet"><p lang="en" dir="ltr">so many reproduced plots! :) <a href="https://t.co/Kc9rpRfsnT">https://t.co/Kc9rpRfsnT</a></p>&mdash; Lukas Heinrich (@lukasheinrich_) <a href="https://twitter.com/lukasheinrich_/status/1230252595303927810?ref_src=twsrc%5Etfw">February 19, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>-->
+
+
+<br clear="all">
```
Comment (henryiii) [inline _posts/2020-02-17-analysis-preservation.md:106]:
```suggestion
<br clear="all"/>
```
(Just looks a little nicer to me to be XML compliant)

File: _posts/2020-02-17-analysis-preservation.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+author: Lukas Heinrich and Kyle Cranmer
+postimage: /assets/images/posts/2020-02-17-preservation-reproduced-plots.jpeg
+postimage-caption: "Participants in [Analysis Preservation Bootcamp](https://indico.cern.ch/event/awesome) showing off their ability to reproduce an LHC analysis. Photo Credit: Samuel Meehan"
+postbanner: /assets/images/posts/2020-02-17-preservation-thumbs-up.jpeg
+summary: "The first [Analysis Preservation Bootcamp](https://indico.cern.ch/event/awesome) was held at CERN from February 17-19, 2020. Thirty graduate students and postdocs learned these new technologies aimed at ensuring reproducibility, streamlining onboarding, and extending the impact of LHC analyses through reuse."
+---
+
+
+One of the cross-cutting themes of IRIS-HEP's Analysis Systems effort is to promote analysis preservation and reuse. This includes streamlining how we capture analysis code as it is being developed as well as creating analysis systems that allow us to efficiently reuse those components. To learn how to properly archive such software-heavy data analyses, 30 young graduate 
+students and postdocs gathered at CERN for a three-day intensitve "[Analysis Preservation
```
Comment (henryiii) [inline _posts/2020-02-17-analysis-preservation.md:13]:
```suggestion
students and postdocs gathered at CERN for a three-day intensive "[Analysis Preservation
```

**Review by henryiii — CHANGES_REQUESTED** [review]

File: _posts/2020-02-17-analysis-preservation.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+is often encoded in software -- from the precise way in which collisions are selected
+to the statistical treatment leading to the final measurement. 
+
+Several technical challenges had to be overcome in order to make this service a reality. The [solution that emerged](http://inspirehep.net/record/1603090) involved leveraging containerization technology used widely in cloud computing as well as workflow technologies. These technologies were abstracted from their high-energy physics origins into the [REANA](http://reanahub.io) product now primarily developed at CERN with contributions from various NSF-funded projects like [DASPOS](https://daspos.crc.nd.edu), [DIANA/HEP](https://diana-hep.org), IRIS-HEP, and [SCAILFIN](https://scailfin.github.io). 
+
+
+On the first two days, students learned to efficiently used modern software engineering
+and cloud computing tools such as version control, continuous integration and Docker
+containers to ensure the analysis code and its development history are properly archived
+and portablly packaged to be used on any cloud computing platform.
```
Comment (henryiii) [inline _posts/2020-02-17-analysis-preservation.md:45]: portablly?

File: _posts/2020-02-17-analysis-preservation.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+On the third and final days, the students then heard from the REANA project to learn
+how to then define computational workflows to preserve the order of analysis steps
+required to carry the analysis from initial event selection all the way to final
+result. Among other things, REANA is used as a computational backend for the
+reinterpretation framework RECAST and thus will play a key role in ongoing summary
+studies at the LHC.
+
+
+With the second Run of the LHC finished and the upgrade for Run-3 underway, some time will pass
+until the four LHC experimenst have collected enough new data to significantly increase
```
Comment (henryiii) [inline _posts/2020-02-17-analysis-preservation.md:55]: experimenst?

File: _posts/2020-02-17-analysis-preservation.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+the sensitivty to Beyond Standard Model phenoemena or the precision of Standard Model
```
Comment (henryiii) [inline _posts/2020-02-17-analysis-preservation.md:56]: sensitivty? phenoemena? (I can't suggest changes anymore now that it's merged)

**Comment by cranmer** [comment]
Oops. I reached out via slack, but looked like you were offline. Feel free
to make edits

**Pull request merged**